File: push/src/main/java/com/xiaomi/mipush/sdk/stat/db/MessageDbHelperFactory.java
package com.xiaomi.mipush.sdk.stat.db;

import android.content.Context;
import com.xiaomi.mipush.sdk.stat.db.base.BaseDbHelper;
import com.xiaomi.mipush.sdk.stat.db.base.BaseDbHelperFactory;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/mipush/sdk/stat/db/MessageDbHelperFactory.class */
public class MessageDbHelperFactory extends BaseDbHelperFactory {
    @Override // com.xiaomi.mipush.sdk.stat.db.base.BaseDbHelperFactory
    public BaseDbHelper getDbHelper(Context context, String str) {
        return MessageDbHelper.newInstance(context, str);
    }
}

File: legacy-runtime/src/main/java/com/xiaomi/mipush/sdk/stat/db/MessageDbHelperFactory.kt
package com.xiaomi.mipush.sdk.stat.db

import android.content.Context
import com.xiaomi.mipush.sdk.stat.db.base.BaseDbHelper
import com.xiaomi.mipush.sdk.stat.db.base.BaseDbHelperFactory

class MessageDbHelperFactory : BaseDbHelperFactory() {
    override fun getDbHelper(context: Context, str: String): BaseDbHelper {
        return MessageDbHelper.newInstance(context, str)
    }
}
